src/main/java/com/logentries/net/AsyncLogger.java

```diff
@@ -666,7 +674,9 @@ public void run() {
 			} catch (InterruptedException e) {
 				// We got interrupted, stop
 				dbg( "Asynchronous socket writer interrupted");
```

Comment:
dbg() will check if debug is enabled, so it wont be a gain (the gain is in avoiding the string concatenation).